README.md

```diff
@@ -16,118 +16,40 @@ on contributing, see our [Contributing guidelines].
 
 GlareDB docs are built using [Just the docs]. The following are needed:
 
-- [Ruby]
-  - Also: [RubyGems] which can typically be installed alongside `ruby`
+- [Ruby] and [RubyGems]
 - GCC (`g++`)
   - `build-essential` APT package for debian and Ubuntu
 - `make`
 
 In addition to the documentation, we have some optional tooling for static
-analysis. The following are needed:
-
-- [Node.js LTS]
+analysis that require [Node.js LTS]. See the `scripts` in `package.json` for
+available tooling.
 
 Once the above are installed, install [jekyll] and [bundler]:
 
-```console
+```shell
 gem install jekyll bundler
 ```
 
-## Getting started
-
-Install project dependencies:
+Next, install project dependencies:
 
-```console
-# From the project root
+```shell
 bundler install
 ```
 
-```console
-# Update project dependencies
-bundler update
-```
-
-Develop locally (`localhost:4000`):
+Finally, a local development server is started on `localhost:4000` with:
 
-```console
-# From root of project
+```shell
 bundle exec jekyll serve
 ```
 
-### Linting
-
-We use `markdownlint` for linting and `prettier` for formatting. To see the
-configurations, check out `.markdownlint-cli2.jsonc` and `.prettierrc.json`
-respectively.
-
-From the root of the project:
-
-```console
-# Checking spelling, formatting, and markdown lints.
-npm run check:all
-
-# Checks spelling
-npm run cspell
-
-# Checks code format
-npm run format
-
-# Attempts to automatically fix any formatting issues
-npm run format:fix
-
-# Checks code adheres to lint rules
-npm run lint
-
-# Attempts to automatically fix any lint violations
-npm run lint:fix
-```
-
-#### Spell checking
-
-See `cspell.json` for spell checking configuration. The `patterns` are regular
-expressions for which spell checking is ignored (example: in code blocks).
-The `words` array is a list of words to ignore.
```

Comment:
I found the linting stuff pretty helpful when working on the re-arch. I know we also have it for the marketing site, but it was nice to have it on hand here. Do you feel strongly about removing it?

Reply:
I don't feel too strongly, no

Reply:
I re-added a simplified version of it!